scripts/make_coverage_map.py

```diff
@@ -181,5 +180,5 @@ def TileBounds(self, tileid):
     #push up 
     with open(args.output_file) as f :
       object_data = f.read()
-      s3_client.put_object(Body=object_data, Bucket=args.ref_speed_bucket, Key= args.ref_bucket_version + "/" + args.output_file)
+      s3_client.put_object(Body=object_data, Bucket=args.ref_speed_bucket, ACL='public-read', Key=args.output_file)
```

Comment:
maybe you want to set the content type to json as well

Reply:
will do